scripts/setup-helper-functions.sh

```diff
@@ -130,6 +131,11 @@ function cmake_install {
   CPU_TARGET="${CPU_TARGET:-avx}"
   COMPILER_FLAGS=$(get_cxx_flags $CPU_TARGET)
 
+  if [[ "$OS" == "Darwin" && "${CPU_TARGET}" == "arm64" ]]; then
+   INSTALL_PREFIX="/opt/homebrew"
```

Comment:
Convention internally is to have homebrew installed on personal home dirs, Can we instead change this to `INSTALL_PREFIX="${INSTALL_PREFIX:-/opt/homebrew}"